pages/account.page.ts

```diff
@@ -1,14 +1,36 @@
-import {Locator, Page } from "@playwright/test";
+import {Locator, Page} from "@playwright/test";
+import { BasePage } from "./base.page";
 import { expect } from '@playwright/test';  
-export class AccountPage {
+export class AccountPage extends BasePage {
     page: Page;
     welcomeMessage: Locator;
+    pageTitle: Locator;
+  navLocator: Locator;
+
     constructor(page: Page) {
+        super(page);
         this.page = page;
         this.welcomeMessage = this.page.getByTestId('page-title');
+        this.pageTitle = this.page.getByTestId('page-title');
+        this.navLocator = this.page.locator('nav');
     }
     
     async verifyWelcomeMessage(expectedName: string): Promise<void> {
         await expect(this.welcomeMessage).toContainText(expectedName);
     }
+      async verifyOnAccountPage(): Promise<void> {
+    await expect(this.page).toHaveURL('https://practicesoftwaretesting.com/account');
```

Comment:
```suggestion
    await expect(this.page).toHaveURL('/account');
```
you already have baseURL set up in playwright config file, so it can be simplified